src/JSHandle.ts

```diff
@@ -151,14 +152,14 @@ export class ElementHandle extends JSHandle {
   }
 
   async _scrollIntoViewIfNeeded(): Promise<void> {
-    const error = await this.evaluate<string | false>(async(element, pageJavascriptEnabled) => {
+    const error = await this.evaluate<Promise<string | false>>(async(element: HTMLElement, pageJavascriptEnabled: boolean) => {
       if (!element.isConnected)
         return 'Node is detached from document';
       if (element.nodeType !== Node.ELEMENT_NODE)
         return 'Node is not of type HTMLElement';
       // force-scroll if page's javascript is disabled.
       if (!pageJavascriptEnabled) {
-        element.scrollIntoView({block: 'center', inline: 'center', behavior: 'instant'});
```

Comment:
This is interesting: TS shouts here because I think `instant` was going to be in the spec but now isn't? So I've ditched this in favour of `auto` which is the default. @mathiasbynens WDYT? I'm not sure if Chrome does still support instant but if it's going/gone from the spec presumably we'll stop supporting it one day.

Reply:
Woah, this is a breaking change. We shouldn't work around TypeScript bugs by changing observable behavior in Puppeteer. Chrome still supports `behavior: 'instant'`. Can we make TypeScript ignore this particular line?

Reply:
@mathiasbynens this isn't a TypeScript bug, but fine, yes we can.

Reply:
(It’s arguably a bug: it doesn’t match real-world browser behavior. In general, when the spec and implementations disagree, then implementations take precedence, IMHO. This is a separate discussion though :])

Reply:
keyword there is "arguably" ;) 😂